Swiftification/String.swift

```diff
@@ -53,6 +53,15 @@ public extension String {
         return result
     }
 
+    /// Returns a value with string inserted at index.
+    @warn_unused_result
+    func insert(string: String?, atIndex index: Int) -> String {
```

Comment:
Ok, but my value isn't a `var`. So, in order to use this the way your'e suggesting, I need to assign it to a var, then mutate it? Or should I have a mutating and non-mutating version?